prov/verbs/src/verbs_srq.c

```diff
@@ -202,41 +224,61 @@ static struct fi_ops fi_ibv_srq_ep_ops = {
 
 
 int fi_ibv_srq_context(struct fid_domain *domain, struct fi_rx_attr *attr,
-		struct fid_ep **rx_ep, void *context)
+		       struct fid_ep **srq_ep, void *context)
 {
 	struct ibv_srq_init_attr srq_init_attr = {};
 	struct fi_ibv_domain *dom;
-	struct fi_ibv_srq_ep *_rx_ep;
+	struct fi_ibv_srq_ep *_srq_ep;
```

Comment:
Can we lose the leading underscore? The fid_ep ** variable can be renamed to srq_ep_fid

Reply:
Yes, I was thinking about that. Thanks. Will do that.